tests/testthat/test-sql-query.R

```diff
@@ -4,3 +4,14 @@ test_that("add_suffixes works if no suffix requested", {
   expect_equal(add_suffixes(c("x", "x"), "y", ""), c("x", "x"))
   expect_equal(add_suffixes(c("x", "y"), "y", ""), c("x", "y"))
 })
+
+test_that("select_query() print method output is as expected", {
+  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
+  expect_equal(
+    paste0(
+      capture.output(print.select_query(mf)),
```

Comment:
You should be able to just use `print()` here, I think?

Reply:
It looks like the `print()` method is overridden so the outputs won't match:

Reply:
library(testthat)
library(dbplyr)

test_that("select_query() print method output is as expected", {
  mf <- select_query(lazy_frame(x = 1, src = simulate_dbi()))
  expect_equal(
    paste0(
      print(dbplyr:::print.select_query(mf)),
      collapse = " "
    ),
    "<SQL SELECT> From:     df Select:   *"
  )
})
#> <SQL SELECT>
#> From:     df
#> Select:   *
#> NULL
#> Error: Test failed: 'select_query() print method output is as expected'
#> * paste0(print(dbplyr:::print.select_query(mf)), collapse = " ") not equal to "<SQL SELECT> From:     df Select:   *".
#> 1/1 mismatches
#> x[1]: ""
#> y[1]: "<SQL SELECT> From:     df Select:   *"